README.md

```diff
@@ -108,6 +102,64 @@ etcd cluster by adding the following to the Helm install command:
 --set etcd.enabled=false --set "operator.controllerSet.spec.dbConnectionURL=jdbc:postgresql://postgres/postgresdb?user=postgresadmin&password=admin123"
 ```
 
+### Configuring storage pool creation
+The piraeus operator installed by helm can be used to create storage pools. Creation is under control of the
```

Comment:
I would prefer it we kept the style consistent - that is, with empty lines after the headings and around the code blocks. I know this is nitpicking and I will merge it anyway, but it would be nice.